.github/workflows/ci.yaml

```diff
@@ -130,27 +130,32 @@ jobs:
               run: choco upgrade llvm
             - name: Setup headless display
               uses: pyvista/setup-headless-display-action@v1
-            - uses: ./.github/actions/install-nodejs
+            - uses: actions/setup-node@v4
+              with:
+                node-version: 20
               id: node-install
+            - uses: pnpm/action-setup@v4.0.0
+              with:
+                version: 9.11.0
             - uses: ./.github/actions/setup-rust
               with:
                   key: x-napi-v2-${{ steps.node-install.outputs.node-version }} # the cache key consists of a manually bumpable version and the node version, as the cached rustc artifacts contain linking information where to find node.lib, which is in a versioned directory.
-            - name: Run npm install
-              working-directory: ./api/node
-              run: npm install --ignore-scripts
+            - name: Run pnpm install
+              working-directory: api/node
```

Comment:
Why did you remove the `./`?

I do not mind you removing that, but then please do it consistently:-)

Reply:
done